src/main/java/com/maxmind/geoip2/model/AnonymousIpResponse.java

```diff
@@ -55,6 +57,43 @@ public AnonymousIpResponse(
         this.network = network;
     }
 
+    @MaxMindDbConstructor
+    public AnonymousIpResponse(
+            @MaxMindDbParameter(name="ip_address") String ipAddress,
+            @MaxMindDbParameter(name="is_anonymous") Boolean isAnonymous,
+            @MaxMindDbParameter(name="is_anonymous_vpn") Boolean isAnonymousVpn,
+            @MaxMindDbParameter(name="is_hosting_provider") Boolean isHostingProvider,
+            @MaxMindDbParameter(name="is_public_proxy") Boolean isPublicProxy,
+            @MaxMindDbParameter(name="is_tor_exit_node") Boolean isTorExitNode,
+            @MaxMindDbParameter(name="network") Network network
+    ) {
+        this(
+            ipAddress,
+            isAnonymous != null ? isAnonymous : false,
+            isAnonymousVpn != null ? isAnonymousVpn : false,
+            isHostingProvider != null ? isHostingProvider : false,
+            isPublicProxy != null ? isPublicProxy : false,
+            isTorExitNode != null ? isTorExitNode : false,
+            network
+        );
+    }
+
+    public AnonymousIpResponse(
```

Comment:
Any reason to keep this and the similar constructors below public?

Reply:
We're calling them from `DatabaseReader` right now, so I think they need to be public unfortunately.

Reply:
Ah, right, I forgot that is in a separate package.